awx/ui_next/src/components/DetailList/Detail.jsx

```diff
@@ -23,9 +23,14 @@ const DetailValue = styled(({ fullWidth, ...props }) => (
     `
     grid-column: 2 / -1;
   `}
+  ${props =>
+    props.missingValue &&
+    `
+    color: #c9190b;
+  `}
 `;
 
-const Detail = ({ label, value, fullWidth }) => {
+const Detail = ({ label, value, fullWidth, missingValue }) => {
```

Comment:
Instead of adding another prop to the `Detail` component for this seldom use case, I think you should use styled components and extend the Detail component to create one for deleted resources.